csharp/Facebook.Yoga/Properties/AssemblyInfo.cs

```diff
@@ -11,22 +11,19 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-// General Information about an assembly is controlled through the following
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
+[assembly: AssemblyTitle("Facebook.Yoga")]
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("Facebook, Inc.")]
 [assembly: AssemblyProduct("Facebook.Yoga")]
-[assembly: AssemblyTrademark("Copyright (c) 2014-present, Facebook, Inc.")]
+[assembly: AssemblyDescription("A subset of CSS's flexbox layout algorithm and box model.")]
+[assembly: AssemblyCopyright("Copyright (c) 2014-present, Facebook, Inc.")]
 
-// Setting ComVisible to false makes the types in this assembly not visible
-// to COM components.  If you need to access a type in this assembly from
-// COM, set the ComVisible attribute to true on that type.
 [assembly: ComVisible(false)]
 
-// The following GUID is for the ID of the typelib if this project is exposed to COM
 [assembly: Guid("75bb7605-e54b-4ede-8f5a-ff1f24464236")]
 
 [assembly: NeutralResourcesLanguage("en")]
 
 [assembly: AssemblyVersion("3.0.0.0")]
+[assembly: AssemblyFileVersion("3.0.0.0")]
```

Comment:
I'm open to any direction on the version to use... Since the name of the project has changed resetting the NuGet version and the corresponding assembly versions to 1.0.0.0 (1.0.0-pre) seems fine to me...